src/SFA.DAS.Provider.PR.Web/Controllers/EmployerDetailsController.cs

```diff
@@ -29,4 +29,31 @@ public async Task<IActionResult> Index([FromRoute] int ukprn, [FromRoute] string
 
         return View(model);
     }
+
+    [Route("{ukprn:int}/employers/{requestid:Guid}", Name = RouteNames.EmployerDetailsByRequestId)]
+    [HttpGet]
+    public async Task<IActionResult> Index([FromRoute] int ukprn, [FromRoute] Guid requestid,
+        CancellationToken cancellationToken)
+    {
+        GetRequestsByRequestIdResponse response;
+        try
+        {
+            response = await _outerApiclient.GetRequestByRequestId(requestid, cancellationToken);
+        }
+        catch
+        {
+            return RedirectToAction("HttpStatusCodeHandler", "Error", new { statusCode = 404 });
+        }
+
+        EmployerDetailsViewModel model = response;
+
+        if (string.IsNullOrEmpty(model.AccountLegalEntityPublicHashedId))
```

Comment:
In the operator we are not assigning this field, so it will always be null? so why do the check?